pipeline/src/node_properties/track_properties.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "tracker", rename_all = "snake_case")]
+pub enum TrackProperties {
+    NvDcf,
```

Comment:
@devarshi talking of future-safety, we probably should use generic names here, so NvDCF -> DCF. The other two are generic already.

Reply:
Yeah, good catch.

Reply:
Yes, good point.